lib/js/__tests__/puppeteer_test.js

```diff
@@ -1,487 +1 @@
-              }));
-        return Jest.afterAllPromise(/* None */0, (function () {
-                      return page[0].close().then((function () {
-                                    return browser[0].close();
-                                  }));
-                    }));
-      }));
-
-describe("ElementHandle", (function () {
-        var browser = [/* () */0];
-        var page = [/* () */0];
-        var elementHandle = [/* () */0];
-        Jest.beforeAllPromise(/* None */0, (function () {
-                return Puppeteer.launch(noSandbox).then((function (res) {
-                                    browser[0] = res;
-                                    return browser[0].newPage();
-                                  })).then((function (res) {
-                                  page[0] = res;
-                                  return page[0].goto("file://" + testPagePath, undefined);
-                                })).then((function () {
-                                return page[0].$("#iframe");
-                              })).then((function (res) {
-                              if (res !== null) {
-                                elementHandle[0] = res;
-                                return Promise.resolve(/* () */0);
-                              } else {
-                                return Promise.reject(Js_exn.raiseError("failed to initial an elementhandle"));
-                              }
-                            }));
-              }));
-        Jest.testPromise(/* None */0, "contentFrame()", (function () {
-                return elementHandle[0].contentFrame().then((function (frame) {
-                              if (frame !== null) {
-                                return Promise.resolve(Jest.ExpectJs[/* toBeTruthy */28](Jest.Expect[/* expect */0](frame)));
-                              } else {
-                                return Promise.reject(Js_exn.raiseError("failed to get frame from contentFrame"));
-                              }
-                            }));
-              }));
-        return Jest.afterAllPromise(/* None */0, (function () {
-                      return page[0].close().then((function () {
-                                    return browser[0].close();
-                                  }));
-                    }));
-      }));
-
-exports.seconds = seconds;
-exports.getElementValueJs = getElementValueJs;
-exports.getLengthOfElementsJs = getLengthOfElementsJs;
-exports.getElementOuterHTMLJs = getElementOuterHTMLJs;
-exports.getElementOuterHTMLJsPromise = getElementOuterHTMLJsPromise;
-exports.fixturesPath = fixturesPath;
-exports.testPagePath = testPagePath;
-exports.testPageJsPath = testPageJsPath;
-exports.testPageCssPath = testPageCssPath;
-exports.testPageContent = testPageContent;
-exports.noSandbox = noSandbox;
-/* getElementValueJs Not a pure module */
+/* This output is empty. Its source's type definitions, externals and/or unused code got optimized away. */
```

Comment:
We lost the generated js for the tests somehow.

Reply:
Oops! Its weird!
Fixed in commit a65cec2